### PR TITLE
add credits and links for haikus

### PR DESCRIPTION
**Pull request by jessnicwelch**
For Instructor Checkout.

### CODE REVIEW

File: _episodes/07-find.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ It is also the name of a very useful command-line program.
 `grep` finds and prints lines in files that match a pattern.
 For our examples,
 we will use a file that contains three haiku taken from a
-1998 competition in *Salon* magazine. For this set of examples,
+[1998 competition](https://web.archive.org/web/19991201042211/http://salon.com/21st/chal/1998/01/26chal.html) in *Salon* magazine (Credit to authors Bill Torcaso, Howard Korder, and Margaret Segall, respectively. See Haiku Error Messsages archived [Page 1](https://web.archive.org/web/20000310061355/http://www.salon.com/21st/chal/1998/02/10chal2.html) and [Page 2](https://web.archive.org/web/20000229135138/http://www.salon.com/21st/chal/1998/02/10chal3.html).). For this set of examples,
```
Comment (bkmgit) [inline _episodes/07-find.md:31]: It is good to breakup long lines where possible
```suggestion
[1998 competition](https://web.archive.org/web/19991201042211/http://salon.com/21st/chal/1998/01/26chal.html)
in *Salon* magazine (Credit to authors Bill Torcaso, Howard Korder, and Margaret Segall, respectively. See 
Haiku Error Messsages archived
[Page 1](https://web.archive.org/web/20000310061355/http://www.salon.com/21st/chal/1998/02/10chal2.html) and
[Page 2](https://web.archive.org/web/20000229135138/http://www.salon.com/21st/chal/1998/02/10chal3.html).). For this set of examples,
```

**Review by bkmgit — CHANGES_REQUESTED** [review]

**Comment by jessnicwelch** [comment]
I believe I integrated the requested changes by @bkmgit. I've never done that before, so let me know if anything else is needed on my end to fulfill this PR. Thanks!

File: _episodes/07-find.md [hunk 2]
```diff
@@ -28,7 +28,11 @@ It is also the name of a very useful command-line program.
 `grep` finds and prints lines in files that match a pattern.
 For our examples,
 we will use a file that contains three haiku taken from a
-[1998 competition](https://web.archive.org/web/19991201042211/http://salon.com/21st/chal/1998/01/26chal.html) in *Salon* magazine (Credit to authors Bill Torcaso, Howard Korder, and Margaret Segall, respectively. See Haiku Error Messsages archived [Page 1](https://web.archive.org/web/20000310061355/http://www.salon.com/21st/chal/1998/02/10chal2.html) and [Page 2](https://web.archive.org/web/20000229135138/http://www.salon.com/21st/chal/1998/02/10chal3.html).). For this set of examples,
+[1998 competition](https://web.archive.org/web/19991201042211/http://salon.com/21st/chal/1998/01/26chal.html)
+in *Salon* magazine (Credit to authors Bill Torcaso, Howard Korder, and Margaret Segall, respectively. See 
+Haiku Error Messsages archived
+[Page 1](https://web.archive.org/web/20000310061355/http://www.salon.com/21st/chal/1998/02/10chal2.html) and
+[Page 2](https://web.archive.org/web/20000229135138/http://www.salon.com/21st/chal/1998/02/10chal3.html).). For this set of examples,
```
Comment (bkmgit) [inline _episodes/07-find.md:35]:
```suggestion
in *Salon* magazine (Credit to authors Bill Torcaso, Howard Korder, and 
Margaret Segall, respectively. See 
Haiku Error Messsages archived
[Page 1](https://web.archive.org/web/20000310061355/http://www.salon.com/21st/chal/1998/02/10chal2.html)
and
[Page 2](https://web.archive.org/web/20000229135138/http://www.salon.com/21st/chal/1998/02/10chal3.html)
.). For this set of examples,
```
Lines were still too long. Build failed. If it seems ok, you should be able to commit these changes directly.

**Review by bkmgit — CHANGES_REQUESTED** [review]

**Comment by jessnicwelch** [comment]
Hmm... looks like the error is `Line(s) too long: 32, 34, 35`. I do not know if this is something I am expected to address.

**Comment by bkmgit** [comment]
Thanks.

**Pull request merged**